forch/forchestrator.py

```diff
@@ -91,60 +87,95 @@ def initialize(self):
-            radius_port = radius_info.get('server_port')
-            secret = radius_info.get('secret')
-            if not (radius_ip and radius_port and secret):
-                LOGGER.warning('Invalid radius_info in config. \
-                               Radius IP: %s; Radius port: %s Secret present: %s',
-                               radius_ip, radius_port, bool(secret))
-                raise ConfigError
-            self._authenticator = Authenticator(radius_ip, radius_port, secret)
-            LOGGER.info('Created Authenticator module with radius IP %s and port %s.',
-                        radius_ip, radius_port)
+        if not radius_info:
+            return
+        radius_ip = radius_info.get('server_ip')
+        radius_port = radius_info.get('server_port')
+        secret = radius_info.get('secret')
+        if not (radius_ip and radius_port and secret):
+            LOGGER.warning('Invalid radius_info in config. \
+                           Radius IP: %s; Radius port: %s Secret present: %s',
+                           radius_ip, radius_port, bool(secret))
+            raise ConfigError
+        self._authenticator = Authenticator(radius_ip, radius_port, secret)
+        LOGGER.info('Created Authenticator module with radius IP %s and port %s.',
+                    radius_ip, radius_port)
 
     def _process_static_device_placement(self):
-        device_info = self._config.get('static_device_info', {})
-        if 'static_device_placement' in device_info:
-            placement_file = os.path.join(
-                os.getenv('FAUCET_CONFIG_DIR'), device_info['static_device_placement'])
-            device_placement_info = yaml_proto(placement_file, DevicesState).device_mac_placements
-            for eth_src, device_placement in device_placement_info.items():
-                self.process_device_placement(eth_src, device_placement)
+        static_placement_file = self._config.get('orchestration', {}).get('static_device_placement')
+        if not static_placement_file:
+            return
+        placement_file = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), static_placement_file)
+        device_placement_info = yaml_proto(placement_file, DevicesState).device_mac_placements
+        for eth_src, device_placement in device_placement_info.items():
+            self.process_device_placement(eth_src, device_placement)
+
+    def _process_static_device_behavior(self):
+        static_behaviors_file = self._config.get('orchestration', {}).get('static_device_behavior')
+        if not static_behaviors_file:
+            return
+        static_behaviors_path = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), static_behaviors_file)
+        devices_state = faucetizer.load_devices_state(static_behaviors_path)
+        for mac, device_behavior in devices_state.device_mac_behaviors.items():
+            self.process_device_behavior(mac, device_behavior)
+
+    def _initialize_faucetizer(self):
```

Comment:
_attempt_faucitizer_initialize or something similar -- since it may not actually initialize it. (same as authenticator class)

Reply:
Now initializing faucetizer based on the result of self._calculate_behavioral_config().